github_metrics/app.py

```
@@ -0,0 +1,1797 @@
+                delivery_id=delivery_id,
+                repository=repository,
+                event_type=event_type,
+                action=action,
+                sender=sender,
+                payload=payload,
+                processing_time_ms=processing_time_ms,
+                status="success",
+                pr_number=pr_number,
+            )
+        except Exception:
+            LOGGER.exception(f"Failed to track webhook event: {delivery_id}")
```

Ruff (0.14.5)

Logging statement uses f-string

(G004)